config/prod/dian-hm-prod-migration-s3.yaml

```diff
@@ -37,6 +44,6 @@ parameters:
 # For CI system (do not change)
 hooks:
   before_launch:
-    - !cmd "curl https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/master/s3-bucket.yaml --create-dirs -o templates/remote/s3-bucket.yaml"
+    - !cmd "wget https://{{stack_group_config.admincentral_cf_bucket}}.s3.us-east-1.amazonaws.com/aws-infra/master/s3-bucket-v2.j2 --create-dirs -o templates/remote/s3-bucket-v2.j2"
```

Comment:
no, this is wrong. please look at the examples in organization-infra repo.  also i would suggest pinning to a specific tag instead of master.

Reply:
This is an outdated version